ewrolemgr.py

```diff
@@ -132,7 +132,7 @@ def setupRoles(client = None, id_server = ""):
 	server_default = None
 ):
 
-	user_data = EwUser(member = member, id_server = server_default)
+	user_data = EwUser(id_user=member.id, id_server = server_default)
```

Comment:
now the function won't work unless you provide the server_default argument

Reply:
sorry about that. discovered it during testing and hadn't pushed the ccommit yet